spec/models/hubstats/pull_request_spec.rb

```diff
@@ -9,6 +9,7 @@ module Hubstats
       expect(pull_request.user_id).to eq(pull[:user][:id])
```

Comment:
You should add/update this spec to verify the behavior with and without the merged_by value set.